tests/torch/data/reference_graphs/quantized/synthetic_model/min.dot

```diff
@@ -1,9 +1,7 @@
 strict digraph  {
 "0 /nncf_model_input_0" [id=0, type=nncf_model_input];
-"1 SymmetricQuantizer/symmetric_quantize_0" [id=1, type=symmetric_quantize];
-"2 TestModel/min_0" [id=2, type=min];
-"3 /nncf_model_output_0" [id=3, type=nncf_model_output];
-"0 /nncf_model_input_0" -> "1 SymmetricQuantizer/symmetric_quantize_0";
-"1 SymmetricQuantizer/symmetric_quantize_0" -> "2 TestModel/min_0";
-"2 TestModel/min_0" -> "3 /nncf_model_output_0";
+"1 TestModel/min_0" [id=1, type=min];
+"2 /nncf_model_output_0" [id=2, type=nncf_model_output];
+"0 /nncf_model_input_0" -> "1 TestModel/min_0";
+"1 TestModel/min_0" -> "2 /nncf_model_output_0";
```

Comment:
There are no quantizers anymore. Why do we need that model?

Reply:
The intention was to test the quantization scheme for atomic operations for Torch. I think that this purpose still exists and correctly reflects the introduced changes by this PR. Moreover, there are several models like this.

Reply:
We do not test the case of keeping the quantizer for this operation now. Add this to another model or update this one to test quantizers' existence and their parameters.

Reply:
I updated the model from min to Conv and min.